### DISCUSSION TITLE
Echo message when server is starting and when they're ready?

### DISCUSSION THREAD

**Discussion by slmjkdbtl** [w0rp/ale#3928] [Q&A - Ask for help with problems]
Some lsp servers in big projects takes a long time to start functioning, is there a way to know when they're ready? It'll be nice to be able to have some feedback on the lsp server state like echo some msg.

**Comment by hsanson** [comment]
This issue is related to this comment https://github.com/dense-analysis/ale/issues/2137#issuecomment-499749636 .

Not sure if things have changed on LSP side to make this possible but from that discussion it seems is not possible to do due to how LSP works.
